app/inithttp.go

```diff
@@ -122,6 +122,9 @@ func (app *App) initHTTP(ctx context.Context) error {
 		// limit max request size
 		maxBodySizeMiddleware(app.cfg.MaxReqBodyBytes),
 
+		// return a 404 if the v1 graphql api is disabled
+		handleServingV1GraphQLRequests(),
```

Comment:
This should be something like `graphQLV1DeprecationMiddleware` or `graphQLV1DisableMiddleware`. Handle implies it's a handler rather than middleware.

Though, it could probably just be inlined too, there's not much logic to it.